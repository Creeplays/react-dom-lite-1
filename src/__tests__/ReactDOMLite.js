import React from 'react';
import { render } from '../';

describe('ReactDOMLite', () => {
  let container;

  beforeEach(() => {
    container = document.createElement('div');
    document.body.appendChild(container);
  });

  afterEach(() => {
    document.body.innerHTML = '';
  });

  it('should render', () => {
    render(<div />, container);

    expect(container).toMatchSnapshot();
  });
});
