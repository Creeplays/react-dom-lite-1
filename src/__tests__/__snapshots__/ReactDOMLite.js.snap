// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReactDOMLite should render 1`] = `
<div>
  <div />
</div>
`;
